src/components/ToDoItem/ToDoItem.js

```diff
@@ -41,30 +50,48 @@ export const ToDoItem = (message) => {
       return;
     }
     toDoMessage.contentEditable = true;
-    editButton.removeEventListener('mouseout', editButtonHoverOff);
     toDoMessage.focus();
     e.preventDefault();
     editButton.style.opacity = 1;
+    const blurClick = () => {
+      const newTaskTitle = toDoMessage.innerHTML;
+      editTaskInStorage(id, newTaskTitle);
+    };
+    toDoMessage.addEventListener('blur', blurClick);
+    editButton.removeEventListener('mouseout', editButtonHoverOff);
   };
 
-  const editToDoButton = () => {
+  const handleEditToDoButtonClick = () => {
     if (!checkbox.checked) {
       toDoMessage.contentEditable = true;
       toDoMessage.focus();
       editButton.style.opacity = 1;
+      const blurClick = () => {
```

Comment:
Why do you have 2 the same blurClick functions which does the same thing? If you want to keep them wrapped in function you can have them inside one 
`handleTaskMessageOnBlur ` OR
`toDoMessage.addEventListener('blur', () => {
  editTaskInStorage(id, toDoMessage.innerHTML)
})`

Reply:
done in 2nd way